src/SDK/Language.php

```diff
@@ -83,4 +83,18 @@ public function getFilters(): array
     {
         return [];
     }
+
+    protected function toUpperCase(string $value): string
```

Comment:
I think the better name would be `toUpperCamelCase` instead? WDYT?

Reply:
I have renamed to `toUpperCaseWords` and `toCamelCase`, I think that's more clear